### PR TITLE
Disable the save button on imported maps.

### PR DESCRIPTION
**Pull request by Fish1**
The user shouldn't be able to save to their profile when they are viewing an imported map.

### CODE REVIEW

File: apps/yapms/src/lib/components/sidebar/sections/savedmaps/SaveMap.svelte [hunk 1]
```diff
@@ -37,13 +38,13 @@
...
 		placeholder="Map Name"
-		disabled={disabled || submitting}
+		disabled={disabled || submitting || $ImportedSVGStore.loaded}
 	/>
 	<div class="tooltip" data-tip="Create">
 		<button
 			class="btn btn-sm btn-primary join-item"
 			on:click={createMap}
-			disabled={disabled || submitting}
+			disabled={disabled || submitting || $ImportedSVGStore.loaded}
```
Comment (Flizzyclone) [inline apps/yapms/src/lib/components/sidebar/sections/savedmaps/SaveMap.svelte]: this condition is bad- $ImportedSVGStore.loaded will still have content if you import a map and then leave that page and navigate to another.

Comment (Flizzyclone) [reply apps/yapms/src/lib/components/sidebar/sections/savedmaps/SaveMap.svelte]: do we lock it based on path??

**Review by Flizzyclone — CHANGES_REQUESTED** [review]

File: apps/yapms/src/lib/components/sidebar/sections/savedmaps/SaveMap.svelte (hunk 1 above)
Comment (Fish1) [reply apps/yapms/src/lib/components/sidebar/sections/savedmaps/SaveMap.svelte]: I think the state should be changed to false if a map isn't loaded.

Comment (Flizzyclone) [reply apps/yapms/src/lib/components/sidebar/sections/savedmaps/SaveMap.svelte]: im cool w/ that, if you wanna have this PR manage that and then use this condition that is good

Comment (Fish1) [reply apps/yapms/src/lib/components/sidebar/sections/savedmaps/SaveMap.svelte]: Eh, your idea looks more clear in the code I think.

**Review by Flizzyclone — APPROVED** [review]

**Pull request merged**